stl/inc/header-units.json

```diff
@@ -6,6 +6,7 @@
     "BuildAsHeaderUnits": [
         // "__msvc_all_public_headers.hpp", // for testing, not production
         "__msvc_chrono.hpp",
+        "__msvc_cxx_stdatomic.hpp",

```

Comment:
Don't we need to remove `stdatomic.h` from this list as well?

Reply:
I don't _think_ so, in that `<stdatomic.h>` should, in fact, still be a header unit.

Having another header-units.json file in vcruntime would be problematic.

Reply:
Won't the header-unit tests fail on GitHub until we update our toolset compiler to one that includes `stdatomic.h` in VCRuntime?

Reply:
...or will it succeed by using the toolset compiler's `stdatomic.h`?

Reply:
I thought the latter, but apparently the former. It has been fixed